static/skins/content/dark/content.css

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
All these skins are part of the node package. Please include them via our `assets/scss/main.scss` (or maybe an additional `tinymce.scss` file incl. in the `main.scss`)
For this file it would look like:
```
@import "node_modules/tinymce/skins/content/dark/content";
```

This should be done for all the css files you pasted here.

For fonts you can include something like this (this one is possibly not exactly right):
```
@font-face {
  font-family: 'tinymce-mobile';
  font-style: normal;
  font-weight: 400;
  src: local('tinymce-mobile'), local('tinymce-mobile-regular'), url(node_modules/tinymce/skins/ui/oxide-dark/fonts/tinymce-mobile.woff) format('woff');
}
```